docs/user-guide/install/rhel.md

```diff
@@ -39,6 +39,13 @@ sudo dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.
 ```
 {: .copy-code}
 
+To ensure proper installation process, execute the following command to install the necessary system fonts:
+
+```bash
+sudo dnf install -y fontconfig dejavu-sans-fonts
```

Comment:
Why do we need fonts in CE? Fonts are necessary for the reporting? If so - add it only for PE and explain why do we need it.